### ISSUE TITLE
Accessibility issues for tables

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1455]
The following accessibility issues were encountered when navigating across tables with a screen reader:
- The fontawesome sort icons are read as 'column # link'. Add some screen reader only text
- All of the `<th>` tags should have a `scope="col"` attribute
- The 'Actions' dropdown menus do not close when the user tabs off of the last link. They should be hidden when the menu loses focus

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'Medium' priority.

**Comment by stephaniesimms** [comment]
I can't test the first two bullets w/o a screenreader. But the third bullet is not fixed. When I tab past the Actions menu it doesn't close.

**Comment by briri** [comment]
@stephaniesimms I vote that we move this one into the backlog. Its likely something wrong with the bootstrap implementation of the dropdown menu or a JS conflict. I don't think we'll have time to resolve in this sprint though.

**Comment by stephaniesimms** [comment]
sounds good @briri - moved to the backlog for now

**Comment by johnpinto1** [comment]
The  fix "All of the <th> tags should have a scope="col" attribute" is not visible to most users, except those with assistive technologies.

The visible change is:
**Current issue with dropdown when tabbing from cell with dropdown**: 
the dropdown remains visible
![Selection_003](https://user-images.githubusercontent.com/8876215/65252498-f2ade680-daf0-11e9-897a-09a8bb7f54f1.png)

This commit closes the dropdown when out of focus.
![Selection_001](https://user-images.githubusercontent.com/8876215/65252693-5506e700-daf1-11e9-817e-82554b0f9f2e.png)
![Selection_002](https://user-images.githubusercontent.com/8876215/65252696-5506e700-daf1-11e9-91b5-b13fd61b4594.png)

**Comment by sjDCC** [comment]
I can tab through to 'Actions' and click enter to open it. If I tab through all options it stays open. There's no navigate away button afaik so the only way to close currently is to select an action

![Capture](https://user-images.githubusercontent.com/6169940/65812141-d974fc00-e1ba-11e9-82d7-8b9a981fc5ae.PNG)

**Comment by johnpinto1** [comment]
@sjDCC The issue above is addrerssed in  https://github.com/DMPRoadmap/roadmap/issues/1450
I have a fix. But need to fix tests for it as they break tests as currently written.

**Comment by johnpinto1** [comment]
@sjDCC @magdalenadrafiova The issue mentioned by @sjDCC in https://github.com/DMPRoadmap/roadmap/issues/1455#issuecomment-536156278 is now fixed. Tabbing away from cell containing dropdown will close it.
Please wait until @xsrust deploys to UAT before testing this.

**Comment by sjDCC** [comment]
Working nicely now - thanks @johnpinto1 Closing out
